src/codex_plus/llm_execution_middleware.py

```
@@ -363,61 +430,356 @@ async def process_request(self, request, path: str):
+                            content=body if body else None,
+                        )
+                    )
+                except Exception as e:
+                    await cleanup_stack.aclose()
+                    logger.error(f"Failed to initiate Cerebras request: {e}")
+                    return JSONResponse({"error": str(e)}, status_code=500)
+
+                if response.status_code >= 400:
+                    try:
+                        error_bytes = await response.aread()
+                    except Exception as read_error:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)